app/controllers/concerns/idv/hybrid_mobile/hybrid_mobile_concern.rb

```diff
@@ -5,10 +5,6 @@ module HybridMobileConcern
 
       include AcuantConcern
 
-      included do
-        before_action :render_404_if_hybrid_mobile_controllers_disabled
```

Comment:
Should we remove [the method](https://github.com/18F/identity-idp/blob/13d1e9b78a7c6cfa44c87c284311165000689ebe/app/controllers/concerns/idv/hybrid_mobile/hybrid_mobile_concern.rb#L48-L50) and associated config as well?

Reply:
Yes on the method, thanks. Removing the config is a bigger undertaking which is [in progress](https://github.com/18F/identity-idp/tree/jmax/LG-9107-remove-fsm-document-capture-hybrid-flow-step).